frontend-react/src/resources/AuthResource.ts

```diff
@@ -26,12 +26,6 @@ export default class AuthResource extends Resource {
     };
 
     static getBaseUrl = () => {
-        if (window.location.origin.includes("localhost"))
-            return "http://localhost:3000";
-        else if (window.location.origin.includes("staging"))
-            return "https://staging.prime.cdc.gov";
-        else if (window.location.origin.includes("test"))
-            return "https://test.reportstream.cdc.gov";
-        else return "https://prime.cdc.gov";
+        return process.env.REACT_APP_BASE_URL;
```

Comment:
Returning `REACT_APP_BACKEND_URL` here would be ideal

Reply:
Oh... is this a mistake?

Backend URL includes /api/ in the path, so the resulting urls built would be wrong unless we correct things.

I'll look into this.